README.md

```diff
@@ -279,8 +279,13 @@ change to adapt it to your specific needs.
     The maximum time in milliseconds the crawler will wait for async listeners.
 * `crawler.userAgent="Node/simplecrawler <version> (https://github.com/cgiffard/node-simplecrawler)"` -
     The user agent the crawler will report.
+* `crawler.decompressResponses=true` -
```

Comment:
I'm actually starting to wonder whether this should even be configurable. Is there a scenario where we'd ever want to switch decompression off? (Ideally I'd like to make sure that the number of configuration options we have to maintain into the distant future is considerably reduced.)